packages/component/src/Styles/StyleSet/AutoResizeTextArea.js

```diff
@@ -0,0 +1,38 @@
+export default function AutoResizeTextArea({ primaryFont }) {
+  return {
+    '&.webchat__auto-resize-textarea': {
+      fontFamily: primaryFont,
+      position: 'relative',
+
+      '& .webchat__auto-resize-textarea__doppelganger': {
+        color: 'transparent',
+        height: '100%',
+        overflowY: 'auto',
+        userSelect: 'none',
+        width: 'inherit',
+        whiteSpace: 'pre-wrap',
+        wordBreak: 'break-word'
+      },
+
+      '& .webchat__auto-resize-textarea__textarea': {
+        appearance: 'none' /* We should tell the user agent we are revamping the style */,
```

Comment:
As breaking change?

Reply:
I would assume idiosyncratic styling is not breaking changes. The "user agent" denotes the browser, not developers.

In the old days, for example, when we style a button, we just set its background color, border, etc. We didn't set `appearance: none` and Safari didn't set the background color for us.

When we are styling any existing HTML form elements (e.g. `<button>`, `<input>`, `<textarea>`, etc), we should always set `appearanace: none`, so the browser don't apply their default stylesheet on.